ecdsa/src/sp1.rs

```diff
@@ -175,11 +198,11 @@ fn recover_ecdsa_unconstrained(sig: &[u8; 65], msg_hash: &[u8; 32]) -> ([u8; 33]
 /// The decompressed public key is 65 bytes long, with 0x04 as the first byte,
 /// and the remaining 64 bytes being the x and y coordinates of the decompressed pubkey in big-endian.
 ///
-/// More details on secp256k1 public key format can be found in the [Bitcoin wiki](https://en.bitcoin.it/wiki/Protocol_documentation#Signatures).
+/// More details on secp256k1/r1 public key format can be found in the [Bitcoin wiki](https://en.bitcoin.it/wiki/Protocol_documentation#Signatures).
```

Comment:
nit: Would recommend finding documentation for the `secpr1` signature that can be put here. Also use `secp256r1` in doc comments to avoid confusion.